### PR TITLE
11.0 merge 3515c55

### PR DESCRIPTION
**Pull request by yelizariev**

### CODE REVIEW

File: portal_event_tickets/README.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+-----------
+* `IT-Projects LLC <https://it-projects.info>`__
+
+Further information
+===================
+
+Demo: http://runbot.it-projects.info/demo/website-addons/10.0
+
+HTML Description: https://apps.odoo.com/apps/modules/10.0/portal_event_tickets/
+
```
Comment (Railia) [inline portal_event_tickets/README.rst]: change version

File: portal_event_tickets/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+{
+    "name": """Customer Event Portal""",
+    "summary": """Allows to customers see their tickets for events at the Portal""",
+    "category": "Marketing",
+    "live_test_url": "http://apps.it-projects.info/shop/product/portal_event_tickets?version=10.0",
+    "images": ["images/banner.jpg"],
+    "version": "10.0.1.0.0",
+    "application": False,
```
Comment (Railia) [inline portal_event_tickets/__manifest__.py]: change version

File: portal_event_tickets/doc/index.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+Installation
+============
+
+* `Install <https://odoo-development.readthedocs.io/en/latest/odoo/usage/install-module.html>`__ this module in a usual way
+
+Configuration
+=============
+
+See documentation for `website_event_attendee_fields <https://apps.odoo.com/apps/modules/10.0/website_event_attendee_fields/>`__
+
```
Comment (Railia) [inline portal_event_tickets/doc/index.rst]: change version

File: website_debranding/__manifest__.py [hunk 4]
```diff
@@ -1,7 +1,6 @@
-
 {
     'name': "Website debranding",
-    'version': '1.0.0',
+    'version': '10.0.1.0.1',
     'author': 'IT-Projects LLC, Ivan Yelizariev',
```
Comment (Railia) [inline website_debranding/__manifest__.py]: change version

File: website_event_attendee_fields/README.rst [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+-----------
+* `IT-Projects LLC <https://it-projects.info>`__
+
+Further information
+===================
+
+Demo: http://runbot.it-projects.info/demo/website-addons/10.0
+
+HTML Description: https://apps.odoo.com/apps/modules/10.0/website_event_attendee_fields/
+
```
Comment (Railia) [inline website_event_attendee_fields/README.rst]: change version

File: website_event_attendee_fields/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,43 @@
+{
+    "name": """Customizable fields for attendees on Events""",
+    "summary": """Do you need more information about attendees than three default fields (name, email, phone)?""",
+    "category": "Marketing",
+    "live_test_url": "http://apps.it-projects.info/shop/product/portal_event_tickets?version=10.0",
+    "images": ["images/banner.jpg"],
+    "version": "10.0.2.0.0",
+    "application": False,
```
Comment (Railia) [inline website_event_attendee_fields/__manifest__.py]: change version

File: website_event_attendee_fields_custom/README.rst [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+-----------
+* `IT-Projects LLC <https://it-projects.info>`__
+
+Further information
+===================
+
+Demo: http://runbot.it-projects.info/demo/website-addons/10.0
+
+HTML Description: https://apps.odoo.com/apps/modules/10.0/website_event_attendee_fields_custom/
+
```
Comment (Railia) [inline website_event_attendee_fields_custom/README.rst]: change version

File: website_event_attendee_fields_custom/__manifest__.py [hunk 8]
```diff
@@ -0,0 +1,41 @@
+{
+    "name": """Attendee registration: Birthdate, Passport, Nationality""",
+    "summary": """Ask information on registration and stores at Partner record""",
+    "category": "Marketing",
+    "live_test_url": "http://apps.it-projects.info/shop/product/portal_event_tickets?version=10.0",
+    "images": ["images/banner.jpg"],
+    "version": "10.0.1.0.0",
+    "application": False,
```
Comment (Railia) [inline website_event_attendee_fields_custom/__manifest__.py]: change version

File: website_event_attendee_fields_custom/doc/index.rst [hunk 9]
```diff
@@ -0,0 +1,5 @@
+=========================================================
+ Attendee registration: Birthdate, Passport, Nationality
+=========================================================
+
+See documentation for `website_event_attendee_fields <https://apps.odoo.com/apps/modules/10.0/website_event_attendee_fields/>`__
```
Comment (Railia) [inline website_event_attendee_fields_custom/doc/index.rst]: change version

File: website_event_attendee_signup/README.rst [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+Demo: http://runbot.it-projects.info/demo/website-addons/10.0
+
+HTML Description: https://apps.odoo.com/apps/modules/10.0/website_event_attendee_signup/
+
+Usage instructions: `<doc/index.rst>`_
+
+Changelog: `<doc/changelog.rst>`_
+
+Tested on Odoo 10.0 c5df78f38bd476bc225778d1c30ff3061e9b23bc
```
Comment (Railia) [inline website_event_attendee_signup/README.rst]: change version

File: website_event_require_login/README.rst [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+
+Demo: http://runbot.it-projects.info/demo/website-addons/10.0
+
+HTML Description: https://apps.odoo.com/apps/modules/10.0/website_event_sale_require_login/
+
+Usage instructions: `<doc/index.rst>`_
+
+Changelog: `<doc/changelog.rst>`_
+
+Tested on Odoo 10.0 bbc226cd6cc17c69979f81831a3da8cc668bf064
```
Comment (Railia) [inline website_event_require_login/README.rst]: change version

File: website_event_require_login/__manifest__.py [hunk 12]
```diff
@@ -0,0 +1,35 @@
+{
+    "name": """Tickets purchasing: force user to sign in / sign up""",
+    "summary": """User registration at your portal is always a plus for marketing""",
+    "category": "Marketing",
+    "live_test_url": "http://apps.it-projects.info/shop/product/portal_event_tickets?version=10.0",
+    "images": ["images/banner.jpg"],
+    "version": "10.0.1.0.0",
+    "application": False,
```
Comment (Railia) [inline website_event_require_login/__manifest__.py]: change version

File: website_sale_cache/README.rst [hunk 13]
```diff
@@ -34,8 +39,21 @@ Maintainers
...
+
+Demo: http://runbot.it-projects.info/demo/website-addons/10.0
+
+HTML Description: https://apps.odoo.com/apps/modules/10.0/website_sale_cache/
+
 Usage instructions: `<doc/index.rst>`_
 
 Changelog: `<doc/changelog.rst>`_
+
+Tested on Odoo 10.0 1ffe85f1cb3defcbf932138e2fc13f3a81b34787
```
Comment (Railia) [inline website_sale_cache/README.rst]: change version

File: website_sale_cache/__openerp__.py [hunk 14]
```diff
@@ -1,10 +1,12 @@
...
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
 {
     "name": """E-commerce Category Cache""",
     "summary": """Use this module to greatly accelerate the loading of a page with a large number of product categories""",
     "category": "Website",
     "images": ['images/websale_cache.png'],
-    "version": "1.0.0",
+    "version": "10.0.1.0.1",
 
     "author": "IT-Projects LLC, Artyom Losev",
```
Comment (Railia) [inline website_sale_cache/__openerp__.py]: change version

File: website_sale_refund/README.rst [hunk 15]
```diff
@@ -0,0 +1,53 @@
...
+
+Demo: http://runbot.it-projects.info/demo/website-addons/10.0
+
+HTML Description: https://apps.odoo.com/apps/modules/10.0/website_sale_refund/
+
+Usage instructions: `<doc/index.rst>`_
+
+Changelog: `<doc/changelog.rst>`_
+
+Tested on Odoo 10.0 60801e537575d08797137d3e34fa2ee492c75f1b
```
Comment (Railia) [inline website_sale_refund/README.rst]: change version

File: website_sale_refund/__manifest__.py [hunk 16]
```diff
@@ -0,0 +1,38 @@
+{
+    "name": """Product swap at eCommerce""",
+    "summary": """Product upgrading / changing feature for your online shop""",
+    "category": "eCommerce",
+    # "live_test_url": "",
+    "images": ["images/banner.jpg"],
+    "version": "10.0.1.0.0",
+    "application": False,
```
Comment (Railia) [inline website_sale_refund/__manifest__.py]: change version

File: website_sale_refund/doc/index.rst [hunk 17]
```diff
@@ -0,0 +1,24 @@
...
+
+Installation
+============
+
+* `Install <https://odoo-development.readthedocs.io/en/latest/odoo/usage/install-module.html>`__ this module in a usual way
+
+You may need one of the following modules, which provide UI to refund some time of products:
+
+* `portal_event_tickets <https://apps.odoo.com/apps/modules/10.0/portal_event_tickets/>`__ -- works with tickets
+
```
Comment (Railia) [inline website_sale_refund/doc/index.rst]: change version

**Comment by yelizariev** [comment]
@Railia those modules are not ported, so I didn't update the versions

**Pull request merged**